dynamic/discovery/discovery.go

```diff
@@ -125,14 +132,29 @@ func (rm *ResourceMap) refresh() {
 				apiResource.Version = gv.Version
 			}
 			gve.resources[apiResource.Name] = apiResource
-			// Remember how to map back from Kind to resource.
+			// Remember how to map back from Kind/subresource to resource.
```

Comment:
I meant for the replacement comment to go here.

```suggestion
			// Remember which resources are subresources, and map the kind to the main resource.
```